paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/samples/ui/shared/MultiToggleButton.kt

```diff
@@ -46,10 +50,20 @@ fun MultiToggleButton(
                 )
             }
 
-            Row(
+            Text(
+                toggleState,
+                color = textColor,
                 modifier = Modifier
-                    .background(backgroundTint)
-                    .padding(vertical = 6.dp, horizontal = 8.dp)
+                    .background(
```

Comment:
Instead of setting the shape in the background, let’s use a `.clip()` call. This makes sure that the ripple effect doesn’t expand beyond the background shape.